src/main/java/com/uber/cadence/converter/JsonDataConverter.java

```diff
@@ -18,13 +18,7 @@
 package com.uber.cadence.converter;
 
 import com.google.common.base.Defaults;
-import com.google.gson.Gson;
-import com.google.gson.GsonBuilder;
-import com.google.gson.JsonArray;
-import com.google.gson.JsonElement;
-import com.google.gson.JsonParser;
-import com.google.gson.TypeAdapter;
-import com.google.gson.TypeAdapterFactory;
+import com.google.gson.*;
```

Comment:
We don't import .* in other places. Please revert.

Reply:
Removed the start import in the following [commit](https://github.com/uber/cadence-java-client/pull/426/commits/58c58bb53cfb53fb37f1de62755936e8532ab726).